Gemfile

```diff
@@ -103,11 +103,24 @@ end
 
 group :development, :test do
   gem 'byebug'
+  gem 'database_cleaner' # Consider removing https://stackoverflow.com/q/49246124
+  gem 'email_spec' # Check usage
   gem 'factory_bot_rails'
   gem 'parallel_tests' # Check usage
+  gem 'pdf-inspector', require: false
   gem 'pry'
+  gem 'rails-controller-testing', require: false # Consider removing and cleaning up controller specs
   gem 'rails-erd' # Check usage
+  gem 'rspec-activemodel-mocks' # Check usage
+  gem 'rspec-collection_matchers'
+  gem 'rspec-html-matchers' # Consider removing and using `have_selector` matchers
   gem 'rspec-rails', '~> 3.9'
+  gem 'shoulda-callback-matchers'
+  gem 'shoulda-matchers', require: false
+  gem 'simplecov', require: false # Check usage
+  gem 'site_prism'
+  gem 'webdrivers', '~> 4.2'
+  gem 'webmock'
 end
```

Comment:
The gemfile changes were because webdrivers wasn't under the development group and bundle install was not updating or installing webdrivers to run specs locally.